format/testfiles/v0_to_v1/keyword_errors.rego

```diff
@@ -7,3 +7,7 @@ contains := 2
 in := 3
 
 every := 4
+
+p {
+	data.foo.contains.bar == 42
```

Comment:
So this won't be formatted to `data.foo["contains"]["bar"]` since the `foo` is not a keyword?

Reply:
It would have been converted to `data.foo["contains"].bar`, but other parts of this module are triggering errors.